.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode10.1
+osx_image: xcode10.2
```

Comment:
Shouldn't we update the `Rakefile` as well?

- Since we'll use Xcode 10.2, we should use iOS 12.2

Other issues in the `Rakefile`:
- The tests on iOS 10.3.1 run twice
- Only the test on iOS 11.4 matches the failing message
```
run_tests('Cauliframework', 'iphonesimulator', 'platform=iOS Simulator,name=iPhone 6,OS=12.1')
build_failed('iPhone 6, iOS 12.0') unless $?.success?
run_tests('Cauliframework', 'iphonesimulator', 'platform=iOS Simulator,name=iPhone 6,OS=11.4')
build_failed('iPhone 6, iOS 11.4') unless $?.success?
run_tests('Cauliframework', 'iphonesimulator', 'platform=iOS Simulator,name=iPhone 6,OS=10.3.1')
build_failed('iPhone 6, iOS 10.0') unless $?.success?
run_tests('Cauliframework', 'iphonesimulator', 'platform=iOS Simulator,name=iPhone 6,OS=10.3.1')
build_failed('iPhone 6, iOS 9.0') unless $?.success?
```

Reply:
Hey @lukaswuerzburger, thanks for the feedback. You are completely right. I updated the Rakefile in that regard and cleaned it up a little.